feature-libs/user/account/components/login-form-oidc/login-form-oidc.component.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { filter, take } from 'rxjs';
+import { OAuthService } from 'angular-oauth2-oidc';
+import { createAuthConfig } from './auth.config';
+import { AuthService, BaseSiteService } from '@spartacus/core';
+
+@Component({
+  selector: 'cx-login-form-oidc',
+  templateUrl: './login-form-oidc.component.html',
+})
+export class LoginFormOidcComponent implements OnInit {
+  protected baseSite: string = '';
+  constructor(
+    protected oauthService: OAuthService,
```

Comment:
Its RECOMMENDED to ue inject() function instead of constructor for Dependency Injection:
refer to: https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines